notebooks/MAST/ultrafaint_dwarf_galaxy_search/ultrafaint_dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<p>1.</p><pre class="ql-syntax" spellcheck="false">Ultra-faint dwarf (UFD) galaxies are a subset of dwarf galaxies that are classified by having one hundred thousand to a few hundred thousand stars, 
</pre><p>The definition of UFD in the introduction needs to be revised. Please add a reference of the first paragraph. There are many articles to discuss about it, but you can refer to the section 1.2 in this review article, Simon (2019) ; in particular, this line in the section.</p><blockquote>we suggest that dwarf galaxies with absolute magnitudes fainter than M_V = −7.7 (L = 10^5 L_sun) be considered UFDs.</blockquote><p> </p><p>2.</p><pre class="ql-syntax" spellcheck="false">All UFD's that have been discovered ... only a few million years after the Big Bang.   
</pre><p>Could you also refer the paper where <code>a few million years</code> was mentioned?</p><p> </p><p>3.</p><p>Could you also add hyperlink to Pan-STARRS (PS1) archive? https://outerspace.stsci.edu/display/PANSTARRS/</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>